src/components/Typography.tsx

```diff
@@ -110,15 +179,16 @@ export function P({style, ...rest}: TextProps) {
     web({
       role: 'paragraph',
     }) || {}
-  const _style = flatten(style)
-  const lineHeight =
-    (_style?.lineHeight || atoms.text_md.lineHeight) *
-    atoms.leading_normal.lineHeight
   return (
     <RNText
       {...attr}
       {...rest}
-      style={[atoms.text_md, t.atoms.text, _style, {lineHeight}]}
+      style={normalizeTextStyles([
+        atoms.text_md,
+        atoms.leading_normal,
```

Comment:
Only the `P` comp has leading applied, all others default to a relative value of 1.0